assets/tests/components/nav/leftNav.test.tsx

```diff
@@ -309,19 +308,6 @@ describe("LeftNav", () => {
     expect(openDrift).toHaveBeenCalled()
   })
 
-  test("clicking About Skate button displays help", async () => {
```

Comment:
I wasn't sure how to test that a new browser window opened, but I also didn't think that this was an unstable enough part of the system that it was worth putting too much effort into figuring out how to write that test.

This is worth a discussion though!

Reply:
It seems like it _could_ be, but I'm struggling a decent amount with getting the types to behave. I wonder if there's an easier way that I'm missing. Would you be up for pairing on it?

Reply:
Happy to pair! Does tomorrow morning work?

Reply:
Right after standup? I'm not usually online before standup

Reply:
We paired and looked into this, because the assertion we want to make is that a new tab opened, and that the browser context doesn't use `window.open` to do so, we couldn't easily find a way to get a spy in to detect the call, so we're going to leave this test out and wait for a future when we figure out/can detect this.